README.md

```diff
@@ -1,49 +1,72 @@
 # Avo Inspector for Go
 
-This repo is under development, check out the [issues](https://github.com/avohq/flutter-avo-inspector/issues) to contribute
+## Avo documentation
 
-If you need any help or guidance don't hesitate to contact me on [Twitter](https://twitter.com/TpoM6oH) or our dev team at dev@avo.app
+This is a quick start guide. For more information about the Inspector project please read [Avo Inspector SDK Reference](https://www.avo.app/docs/implementation/avo-inspector-sdk-reference) and the [Avo Inspector Setup Guide](https://www.avo.app/docs/implementation/setup-inspector-sdk).
 
-# Intro
+## Installation
 
-At Avo we make sure that your product analytics functions properly. 
 
-One of the tools we have for that is the Avo Inspector. 
-
-It observes the analytics calls of an app, e.g. 
-```
-MyTracker.track("Login", {
-  "userId": 1337,
-  "emailAddress": "jane.doe@avo.app",
-  "productId": 45,
-  "revenue": 15.99,
-  "timestamp": 1579263014,
-  "deviceId": "2500-11ec-9621"
-});
 ```
-You can call Avo Inspector SDK with the same parameters as the tracking calls 
+    go get github.com/avohq/go-avo-inspector
 ```
-AvoInspector.trackSchemaFromEvent("Login", { 
-  "userId": 1337, "emailAddress": "jane.doe@avo.app",
-  "productId": 45, "revenue": 15.99,
-  "timestamp": 1579263014, "deviceId": "2500-11ec-9621"
-});
+
+## Initialization
+
+Obtain the API key at [Avo.app](https://www.avo.app/welcome)
+
+```go
+import (
+	avoinspector "github.com/avohq/go-avo-inspector"
+)
+
+avoInspector, err := avoinspector.NewAvoInspector(
+    "...", // Your API key obtained in the Avo workspace
+    avoinspector.Dev, // or avoinspector.Stafging, avoinspector.Prod
+    "1.0", // App version
+    "my app", // App name
+    )
+
 ```
-and it will extract the event schema and send that schema to Avo servers for inspection.
 
-The resulting payload sent to Avo will look like this:
+## Enabling logs
+
+Logs are enabled by default in the dev mode and disabled in prod mode.
+
+```go
+avoInspector.ShouldLog(true)
 ```
-{
-  "userId": "int",
-  "emailAddress": "string",
-  "productId": "int",
-  "revenue": "float",
-  "timestamp": "int",
-  "deviceId": "string"
-}
+
+## Sending event schemas
+
+Whenever you send a tracking event, also call the following method:
+
+Read more in the [Avo documentation](https://www.avo.app/docs/implementation/devs-101#inspecting-events)
+
+### 1.
+
+This method gets actual tracking event parameters, extracts schema automatically and sends it to the Avo Inspector backend.
+It is the easiest way to use the library, just call this method at the same place you call your analytics tools' track methods with the same parameters.
+
+```go
+result, err := avoInspector.TrackSchemaFromEvent("Test Event", map[string]interface{}{
```

Comment:
Is the last part a bit strange? Missing some spaces? `map[string]interface{}{`

Reply:
That's how we (me and my AI friend) write go code 😄